### PR TITLE
Fix spacing between icon and "Output" button

### PR DESCRIPTION
**Pull request by YeldhamDev**
| Before: | After: |
|---|---|
| Screenshot_20230725_191326 | Screenshot_20230725_181745 |

### CODE REVIEW

**Comment by YeldhamDev** [comment]
The CI error seems unrelated to the PR.

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -7824,6 +7824,8 @@ EditorNode::EditorNode() {
 
 	log = memnew(EditorLog);
 	Button *output_button = add_bottom_panel_item(TTR("Output"), log);
+	// Add separation for the warning/error icon that is displayed later.
+	output_button->add_theme_constant_override("h_separation", 6 * EDSCALE);
```
Comment (YuriSizov) [inline editor/editor_node.cpp:7828]: We should move this to the editor theme and reuse it between here and the debugger.

Comment (YeldhamDev) [reply editor/editor_node.cpp]: Done.

File: editor/editor_themes.cpp [hunk 2]
```diff
@@ -1453,6 +1453,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_stylebox("panel", "TabContainer", style_content_panel);
 
 	// Bottom panel.
+	theme->set_constant("h_separation", "BottomPanelButton", 6 * EDSCALE);
```
Comment (YuriSizov) [inline editor/editor_themes.cpp:1456]:
```suggestion
	theme->set_type_variation("BottomPanelButton", "Button");
	theme->set_constant("h_separation", "BottomPanelButton", 6 * EDSCALE);
```

Using type variations for this is a good idea, but we should define it properly :)

**Review by YuriSizov — APPROVED** [review]

**Review by Calinou — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**